plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/starlark/completion/lookups/StarlarkParameterLookupElement.kt

```diff
@@ -0,0 +1,13 @@
+package org.jetbrains.bazel.languages.starlark.completion.lookups
+
+import com.intellij.codeInsight.lookup.LookupElementPresentation
+import org.jetbrains.bazel.languages.starlark.psi.functions.StarlarkParameter
+
+class StarlarkParameterLookupElement(element: StarlarkParameter) : StarlarkNamedLookupElement(element, StarlarkQuote.UNQUOTED) {
+  override fun getLookupString(): String = "$name = "
```

Comment:
**suggestion:** Reconsider the fixed format for parameter lookup strings

The current implementation assumes all parameters require a space and equals sign. Consider a more flexible approach that can adapt to different parameter types or usage contexts.